pages/mesosphere/dcos/2.0/release-notes/2.0.5/index.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 2.0.5
+title: Release notes for 2.0.5
+menuWeight: 0
+excerpt: Release notes for DC/OS 2.0.5, including Open Source attribution, and version policy.
+---
+DC/OS&trade; 2.0.5 was released on ???? 2020.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/2.0.5/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="https://downloads.mesosphere.com/dcos-enterprise/stable/2.0.5/dcos_generate_config.ee.sh"]Download DC/OS Enterprise* [/button]
+
+New customers, contact your sales representative or <a href="mailto:sales@mesosphere.io">sales@mesosphere.io</a> before attempting to download and install DC/OS Enterprise.
+
+# Release Summary
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment.
+
+# DC/OS 
+
+## Components
+
+DC/OS 2.0.5 includes the following component versions:
+
+- Apache&reg; Mesos&reg; 1.9.1-dev
+- OpenSSL 1.1.1g
+
+### DC/OS Fixed and Improved Issues
```

Comment:
Missing items:
1. D2IQ-68394 COPS-6128: "Fixed a problem where Zookeeper log messages were not being forwarded to syslog."
2. (No JIRA), but the PR is here: https://github.com/dcos/dcos/pull/7328
   Suggested wording: "Fixed a problem where the DC/OS UI sometimes did not report the correct DC/OS version due to browser caching".
3. (D2IQ-68800) PR is here: https://github.com/dcos/dcos/pull/7341  
   Suggested wording: "Fixed a problem where frameworks could interfere with Marathon pods by launching tasks on resources reserved to Marathon."

Reply:
@cneth see below:
1. added to release notes but used previously written description (2.1 release notes)
2. Waiting on development to add a JIRA before I can add this. 
3. This item is not a COPS ticket, and @vinodkone has instructed to only add COPS tickets for DCOS at this time. Will need to wait for clarification from him.

Reply:
@cneth The only COPS tickets recommendation was for minor releases only. For patch releases, I think we should include all relevant JIRAs.

Reply:
created https://jira.d2iq.com/browse/D2IQ-69682 for 2.

Reply:
Ok, items added.